### PR TITLE
Split the bulk of VerifyAccountView off into a mixin

### PR DESCRIPTION
**Pull request by adam-thomas**
Turns out DUM had a `VerifyAccountView` in it after all!  This PR splits most of it off into a mixin in order to reimplement the non-API version I added the other day.

@incuna/backend review please?

### CODE REVIEW

File: user_management/utils/views.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core import signing
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.exceptions import PermissionDenied
+
+from user_management.api.exceptions import InvalidExpiredToken
+
+
+class VerifyAccountViewMixin():
```
Comment (Ian-Foote) [inline user_management/utils/views.py]: Subclass `object` for python 2.

Comment (adam-thomas) [inline user_management/utils/views.py]: joy

**Comment by adam-thomas** [comment]
@incuna/backend any more comments? :)

**Comment by adam-thomas** [comment]
Thanks :)

**Pull request merged**